templates/terraform/monster-infrastructure/provider.tf

```
@@ -0,0 +1,11 @@
+provider google {
+  alias = "target"
```

```suggestion
  alias = "command-center"
```
For ultimate clarity